src/Compute/Compute/Manual/NewAzureRmVmssConfigCommand.cs

```diff
@@ -29,6 +29,7 @@ public partial class NewAzureRmVmssConfigCommand : Microsoft.Azure.Commands.Reso
         [Parameter(
             Mandatory = true,
             ParameterSetName = "AssignIdentityParameterSet")]
+        [CmdletParameterBreakingChange("AssignIdentity", ChangeDescription = "'IdentityType' parameter with 'SystemAssigned' value will be used instead of 'AssignIdentity'")]
```

Comment:
Why do you add a new breaking change here?

Reply:
It is a long over-due breaking change.  AssignIdentity is changed to IdentityType with SystemAssigned.   I will revert it now, and will add this breaking change warning message in the next release.

Reply:
It's better to submit these attributes after this release. This may confuse us when we check the breaking changes.